src/components/Uploader/styles.js

```diff
@@ -81,10 +81,12 @@ export const PreviewActions = styled.div`
   background: ${theme.color.element.overlay};
   padding: 0 ${theme.space.m};
   text-align: right;
-  opacity: 0;
```

Comment:
https://user-images.githubusercontent.com/512823/192714456-86dd046f-c48d-45af-b70a-715f559ed791.mov

That doesn't really work because now the whole thing is black when you don't have your mouse on it. I would make the background transparent there, and add a rule to make it black on hover. And maybe the `transition` rule here is not useful anymore.

Reply:
I'm stupid I was using a black screenshot to do my test, I haven't noticed 🤦‍♀️ changing now

Reply:
That's an original reason to not see a problem 😄

Reply:
The way this component was designed was to show the bin icon on hover, we should be able to make the button white on hover:
<img width="1504" alt="image" src="https://user-images.githubusercontent.com/4759751/192737141-f34a42af-df40-4591-8934-3a9501be4147.png">

Reply:
It's probably a lot nicer to change the design to always show the bin icon (not only on hover) what do you guys think? - to do that we would need to add a dark background to the button and make the icon white, so that it is visible on any images dark/light.

Reply:
I've asked Pierre, he told me to keep it blue  🤷‍♀️
What do you think about this version? The trash icon will always be visible

Reply:
I think that is great but as mentioned before, if we always show the button we will need to add a background to the button as it will not display nicely on darker images.
<img width="834" alt="image" src="https://user-images.githubusercontent.com/4759751/192748999-03b70fa1-4bc7-40d4-b473-1b38beade92b.png">

Reply:
Something like this, we can maybe check with Pierre what colors he prefers:
<img width="1148" alt="image" src="https://user-images.githubusercontent.com/4759751/192749353-0ac71e45-f835-4b82-b38c-05f179d15dbd.png">

Reply:
On my opinion, it doesn't look great with a background 😝 I'm sending the different pictures to Pierre to let him choose the best option

Reply:
Pierre prefers with a white background, i'm changing it :)

Reply:
But that will be bad UX - what if the image that is being uploaded is the same blue color? we will not see the icon. 
Maybe Pierre can come up with a nicer background color, maybe one with a bit of transparency.

Reply:
Yes true... Let's goo with a background